docs/views.rst

```diff
@@ -28,7 +25,12 @@ created function:
         Pattern('hello', hello)
     ]
 
-Now your bot is able to return a message when receiving a `hello` message.
+Now your bot is able to respond a `hello` message.
+
+Sync views
+^^^^^^^^^^
+
+In the Bottery documentation, all the views definitions uses the `async` keyword. That doesn't mean that every view on your bot must be defined with `asyncio` syntax. You just need to be careful when creating views as normal functions, sice you can accidently block your bot until executing this view.
```

Comment:
> since you can accidentlly block your bot until executing this view.

perhaps mentioning why that would happen:

> since you can accidently block your bot while executing this view by making a block operation, for example a request to an external website that takes too long.